packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-styles.spec.ts

```diff
@@ -1,14 +1,13 @@
 import { getComponentCss } from './segmented-control-item-styles';
 
-xdescribe('getComponentCss()', () => {
+describe('getComponentCss()', () => {
   it.each<Parameters<typeof getComponentCss>>([
-    [false, false, 'background-default', 'light'],
-    [true, false, 'background-default', 'light'],
-    [true, true, 'background-default', 'light'],
-    [false, true, 'background-default', 'light'],
-    [false, false, 'background-surface', 'light'],
-    [false, false, 'background-default', 'dark'],
-    [false, false, 'background-surface', 'dark'],
+    [false, false, true, 'light'],
+    [true, false, true, 'light'],
+    [true, true, true, 'light'],
+    [false, true, true, 'light'],
+    [false, false, false, 'light'],
+    [false, false, false, 'dark'],
   ])('should return correct css for isDisabled: %s, isSelected: %s, bgColor: %s and theme: %s', (...args) => {
```

Comment:
Message still does not fit the values provided. It should be `should return correct css for isDisabled: %s, isSelected: %s, hasIcon: %s and theme: %s`

Reply:
Fixed it